### PR TITLE
docs(all) Patch files to docs/usage-playground

### PR DESCRIPTION
**Pull request by YipingRuan**
## Description

Just to put the shared changes to all the remaining components.
## Markup:

<details>

```vue
// Paste your FULL Playground.vue here
```
</details>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:

- [ ] The PR title is no longer than 64 characters.
- [ ] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and breaking changes).
- [ ] My code follows the code style of this project.
- [ ] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)
- [ ] I've added new examples to the kitchen (applies to new features and breaking changes in core library)

### CODE REVIEW

**Comment by YipingRuan** [comment]
````python
import sys, json, re, os
from json import JSONDecoder
from collections import OrderedDict

repoPath = 'C:/Repos/GitHub/vuetify'
# print ("sys.argv %s" % (sys.argv[1]))
name = sys.argv[1]  # Calendars, DataIterators
name_kebab_case = re.sub(r'(?<!^)(?=[A-Z])', '-', name).lower()
jsonDecoder = JSONDecoder(object_pairs_hook=OrderedDict)

def patch_component_json(c):
    # C:\Repos\GitHub\vuetify\packages\docs\src\data\pages\components\DataIterators.json
    # packages/docs/src/data/pages/components/xxx.json
    path = repoPath + '/packages/docs/src/data/pages/components/' + c + '.json'
    with open(path) as jsonFile:
        content = jsonFile.read()
        content = content.replace('"type": "usage",', '"type": "usage-new",')
        content = content.replace('"value": "usage"', '"value": {\n            "booleans": []\n          }')
    with open(path, 'w') as outfile:
        outfile.write(content)

    print("Patched: " + path)
    pass

def delete_usage_vue(n):
    # Delete "packages/docs/src/examples/xxx/usage.vue"
    t = repoPath + '/packages/docs/src/examples/' + n + '/usage.vue'
    os.remove(t)
    print("Deleted " + t)
    pass

def patch_component_json2(c):
    # packages/docs/src/lang/en/components/DataIterators.json
    path = repoPath + '/packages/docs/src/lang/en/components/' + c + '.json'
    with open(path) as jsonFile:
        content = jsonFile.read()
        data = jsonDecoder.decode(content)
        desc = data['examples']['usage']['desc']
        content = content.replace('"examples": {', '"usageText": "' + desc + '",\n  "examples": {')
        # data = jsonDecoder.decode(jsonFile_usageText)
        # del data['examples']['usage']
        content = re.sub(r'\n    "usage": {.+?    },', '', content, flags=re.DOTALL)
    with open(path, 'w') as outfile:
        outfile.write(content)

    print("Patched: " + path)
    pass

def add_vue(n):
    # packages/docs/src/usages/data-iterators.vue
    path = repoPath + '/packages/docs/src/usages/' + n + '.vue'
    with open(path, 'w') as outfile:
        text = """<template>
  <v-calendar v-bind="attrs" />
</template>

<script>
  import Usage from './usage'

  export default {
    mixins: [Usage],
  }
</script>
"""
        outfile.write(text)
    print("Created: " + path)
    pass

patch_component_json(name)
delete_usage_vue(name_kebab_case)
patch_component_json2(name_kebab_case)
add_vue(name_kebab_case)
`````

File: packages/docs/src/lang/en/components/Lazy.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
   "heading": "# Lazy loading",
   "headingText": "The `v-lazy` component is used to dynamically load components based upon an elements visiblity.",
   "lazyAlert": "The `v-lazy` component uses the [v-intersect](/directives/intersect) directive which requires a [Polyfill](/directives/intersect#polyfill) in order to work on IE11 / Safari. It's possible on some iOS versions will also require the use of this polyfill.",
+  "usageText": "The `v-lazy` component by default will not render its contents until it has been intersected. Scroll down and watch the element render as you go past it.",
   "examples": {
     "usage": {
       "desc": "The `v-lazy` component by default will not render its contents until it has been intersected. Scroll down and watch the element render as you go past it."
```
Comment (jacekkarczmarczyk) [inline packages/docs/src/lang/en/components/Lazy.json]: Forgot to remove?

Comment (YipingRuan) [reply packages/docs/src/lang/en/components/Lazy.json]: This one didn't end with comma and survived the replacement.
So should leave examples {} or remove examples at all?

**Comment by YipingRuan** [comment]
> * [ ]  Seems there's about 35 components/pages that were excluded
> * [ ]  should probably migrate eo-UY keys to prevent loss of crowdin translations

Seems the rest didn't follow the common style and failed to match the lookup. 
Would it be okay we finish the existing one first?

Anyway what to do now? If there is git operation needed someone please click some button...

**Comment by MajesticPotatoe** [comment]
> Anyway what to do now?

Well, would be silly to only do half of them... So should probably finish the rest of them.

**Comment by YipingRuan** [comment]
Okay give me sometime to clear the rest.

Still have no idea for eo-YU key issue.

**Comment by MajesticPotatoe** [comment]
> Still have no idea for eo-YU key issue.

its essentially the same thing you are doing in lang/en, moving the example usage desc, out of examples and changing it to usageText.

**Pull request merged**